### ISSUE TITLE
Component - Alternative header (w/image)

### ISSUE THREAD

**Issue by twblack88** [makerdao/community-portal#29]
For the learn pages, this is a good introduction to topic option. Might leverage the structure from #25 

[Extra context](https://www.notion.so/Components-e4455d1797f74c48aae3019692a39413#e479a9b64c304c36b2cfb50329243bf6)

[Frame with that header](https://www.figma.com/file/mVblaaFzkqracWycXl9czh/Maker-comm-dev?node-id=486%3A632)

**Comment by as-dr** [comment]
@MaximumCrash My hunch would be to do the same as I proposed for the info blocks, i.e.

Have two containers, one for the left column (the written content) and one for the right column ( the image content) that are themselves contained inside a `wrapper` element (ideally a `ul` to provide better screen reader support).

1. Use `flex-wrap` to let them wrap to new lines as screen real estate reduces. [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-wrap)
1. Use `order` to ensure the images and written content are ordered in the desired manner (the accessibility issues around this are mitigated since we would be reordering an image and a piece of written copy, rather than two pieces of written copy) [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/order)

**Comment by MaximumCrash** [comment]
@as-dr The problem for this on mobile is that because a content creator can write the component any way they want there could be inconsistencies with how it'd render on mobile. 

For example, this looks fine on Desktop and tablet. 
![Screen Shot 2020-06-11 at 12 34 32 PM](https://user-images.githubusercontent.com/9157341/84414553-ebc78680-abdf-11ea-86d4-0623868d819c.png)


But once you start talking about Mobile, from a design perspective it would "generally" look better for the Image to come first before the content. 

Because this is only written once in an MDX file:
```
<AlternativeImage >
Maker makes decisions... ect. ect.
![makerlogo](path/to/makerlogo/)
</AlternativeImage>
```
OR
```
<AlternativeImage image="path/to/makerlogo/" imgLast>
Maker makes decisions... ect. ect.
</AlternativeImage>
```

The above works fine on Desktop but I don't think folks are going write mobile specific MDX files to meet 2 design cases.

**Comment by as-dr** [comment]
It's not possible to hardcode (hardcore aka "take this decision out of the hands of the person using the component") the image so that it adopts an `order: 1` style conditionally, when the viewport is < 480px wide?

e.g.

pseudocode:
```
<AlternativeHeader>
  <AlternativeHeaderImage [viewport === < 480px && order: 1]  />
  <AlternativeHeaderCopy>
    Blah blah blah
  </AlternativeHeaderCopy>
</AlternativeHeader>
```
The "natural" approach to establish this convention w/o the `order` style is (and you know this) for the image in the AlternativeHeader component is always placed on the left side, so when the container containing both `img` and `copy` wrap, it's on the top.

**Comment by MaximumCrash** [comment]
@as-dr Yep that approach would be possible, but I can't put my foot down to make the call. I'll leave that to the team to come to a decision with our new designer.

**Comment by MaximumCrash** [comment]
@twblack88 this component and Info blocks are carbon copies of each other. I will compile them into 1 component (InfoBlock).

**Comment by MaximumCrash** [comment]
This issue is closed by #48
